### ISSUE TITLE
Cannot install language server 0.9

### ISSUE THREAD

**Issue by hazcod** [hashicorp/vscode-terraform#221]
VSCode Version: 1.38.1
Extension version 1.4.0

```
Command 'Terraform: Install/Update Language Server' resulted in an error.
```

**Comment by davidtom** [comment]
@hazcod how were you able to fix this issue? I'm running into the same problem right now.

**Comment by hazcod** [comment]
@davidtom I blamed it on https://github.com/mauve/vscode-terraform/issues/157 , but not sure since there is no response there.. Reopening.

**Comment by davidtom** [comment]
@hazcod Oh I see, thanks for linking me there. I haven't yet read through that thread, but will try to at some point today. If I'm able to get mine up and running I'll let you know!

**Comment by jeffb4** [comment]
Do you see any diagnostic outputs? Check the Outputs -> Terraform and/or Outputs -> Language Server output in vscode

**Comment by davido-bs** [comment]
I'm getting similar issues with the AWS provider in particular. The Language Server outputs basically just state:
```
2019-10-21T17:01:20.331-0700 [DEBUG] plugin: plugin process exited: path=/home/davido/.vscode/extensions/mauve.terraform-1.4.0/lspbin/terraform-provider-aws_v2.33.0_x4 pid=29568
2019-10-21T17:01:20.331-0700 [DEBUG] plugin: plugin exited
```
And it seems like it goes in an infinite loop of killing and restarting that process. The initial error I received was `Failed to download common providers: Error: ETXTBSY: text file is busy, copyfile '/home/davido/.vscode/extensions/mauve.terraform-1.4.0/lspbin/.terraform/plugins/linux_amd64/terraform-provider-aws_v2.33.0_x4' -> '/home/davido/.vscode/extensions/mauve.terraform-1.4.0/lspbin/terraform-provider-aws_v2.33.0_x4'`.

**Comment by paultyng** [comment]
We just released [v2.0.0-rc.1](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) of the extension. The main features include:

* Added syntax support for 0.12
* Added [terraform-ls](https://github.com/hashicorp/terraform-ls) usage by default (currently on 0.3.0, which offers basic provider code completion)

You can find additional information and specifics in the [release notes](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) and [CHANGELOG](https://github.com/hashicorp/vscode-terraform/blob/master/CHANGELOG.md).

With this release we expect that many of the prior issues and PRs are no longer relevant or have been addressed, and are therefore being closed. If you feel the action taken on an issue or PR is in error, please comment as such and we can figure out the appropriate way to address it.

We plan to add the final 2.0.0 release to the marketplace soon, but are actively seeking your feedback now on the release candidates. You can download the .vsix from the releases page and [manually install it in VS Code](https://code.visualstudio.com/docs/editor/extension-gallery#_install-from-a-vsix) to try it out.
